content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -0,0 +1,109 @@
+weight: 70
+---
+
+{{< warning >}}
+
+This document describes **Alpha** quality features. They have been exposed *only* to enable further testing
+and evaluation across the community.
+
+The functionality described here is NOT considered stable. No performance or stability guarantees are provided
+at this time.
+
+{{< /warning>}}
+
+As of the 1.3 release, Istio provides experimental support for the generation of service-level HTTP metrics
+directly in Envoy. This enables operators to continue to monitor their service meshes using Istio-provided
+tooling, without requiring the use of Istio's Mixer. This support is the first step towards a new
+extensibility architecture for Istio, which will put Mixer-like functionality directly in the sidecar proxies.
+
+The in-proxy generation of service-level metrics is a drop-in replacement for the HTTP metrics currently
+generated by Mixer, namely: `istio_requests_total`, `istio_request_duration_*` and `istio_request_size`.
+
+## Enable Service-level Metrics Generation in Envoy
+
+To enable the generation of service-level metrics in Envoy, execute the following steps:
+
+1. Disable calls to `istio-telemetry` in the mesh
+
+    In order to prevent duplicate telemetry generation, turn off the use of the `istio-telemetry` service
+    for telemetry generation.
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system --set mixer.telemetry.enabled=false --set mixer.policy.enabled=false
+    {{< /text >}}
+
+    Alternatively, comment out `mixerCheckServer` and `mixerReportServer` from the mesh config
+    (the `istio` config map).
+
+1. Enable the metadata exchange filter
+
+    For the generation of service-level metrics, proxies must exchange workload metadata. This exchange
+    is handled by a custom filter.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f https://github.com/istio/proxy/blob/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
+    {{< /text >}}
+
+1. Enable the custom stats filter
+
+    A custom filter must also be applied to actually generate the service-level metrics.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/stats_filter.yaml
+    {{< /text >}}
+
+1. See the results
+
+    Visit Grafana "Istio Mesh" dashboard. It will display the same telemetry as before -- without
+    any requests flowing through Istio's Mixer.
+
+## Differences with Mixer-based generation
+
+There are some small differences between the in-proxy generation of the service-level metrics and
+the Mixer-based generation in the 1.3.0 release. Full feature-parity is a requirement before this
+functionality can be considered stable.
+
+In the meantime, please be aware of these differences:
+
+1. The latency metric uses a new name and is recorded in different units. The new metric is called
```

Comment:
```suggestion
- The `istio_request_duration_seconds` latency metric has the new name: `istio_request_duration_milliseconds`. 
  The new metric uses milliseconds instead of seconds. We updated the Grafana dashboards to
  account for these changes.
```